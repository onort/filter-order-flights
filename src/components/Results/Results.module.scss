@import "../../styles/main.scss";

.container {
  margin-bottom: map-get($spacers, large);
}

.text {
  margin: 0 0 2rem;
}
