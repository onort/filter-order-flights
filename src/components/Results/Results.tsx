import React, { Component } from "react"

import styles from "./Results.module.scss"

import { Card, ResultsTable } from "../"

class Results extends Component {
  public render() {
    return (
      <main className={styles.container}>
        <Card>
          <ResultsTable />
        </Card>
      </main>
    )
  }
}

export default Results
