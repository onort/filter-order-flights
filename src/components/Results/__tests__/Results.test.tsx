/* tslint:disable jsx-key */

import React from "react"
import { mount, shallow } from "enzyme"

import Results from "../"
import { Card, ResultsTable } from "../../"

describe("<Results />", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Results data={[]} />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.html()).toMatchSnapshot()
  })

  it("should have correct children", () => {
    const wrapper = mount(<Results data={[]} />)
    expect(
      wrapper.containsAllMatchingElements([
        <Card />,
        <ResultsTable data={[]} />
      ])
    )
    wrapper.unmount()
  })
})
