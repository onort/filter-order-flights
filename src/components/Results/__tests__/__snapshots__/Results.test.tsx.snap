// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Results /> should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Results
    data={Array []}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Card
        className="tableWrapper"
      >
        <ResultsTable
          data={Array []}
        />
      </Card>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ResultsTable
          data={Array []}
        />,
        "className": "tableWrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "data": Array [],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Card
          className="tableWrapper"
        >
          <ResultsTable
            data={Array []}
          />
        </Card>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <ResultsTable
            data={Array []}
          />,
          "className": "tableWrapper",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "data": Array [],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;

exports[`<Results /> should match snapshot 2`] = `"<main class=\\"container\\"><div class=\\"container tableWrapper\\"><div class=\\"container noResult\\"><span>Aradığınız kriterlere uygun uçuş bulunamadı.</span></div></div></main>"`;
